src/libcore/ops/drop.rs

```diff
@@ -18,63 +18,9 @@
 ///
 /// ## Implementing `Drop`
 ///
-/// When a value goes out of scope, the compiler automatically insert a call to `Drop::drop` on that value. For example,
-///
-/// ```rust
-/// struct HasDrop;
-///
-/// impl Drop for HasDrop {
-///     fn drop(&mut self) {
-///         println!("Dropping!");
-///     }
-/// }
-///
-/// fn main() {
-///     let _x = HasDrop;
-/// }
-/// ```
-///
-/// Here, `main` will print `Dropping!`. In other words, the compiler generates code that
-/// kind of looks like this:
-///
-/// ```rust,compile_fail,E0040
-/// # struct HasDrop;
-/// #
-/// # impl Drop for HasDrop {
-/// #     fn drop(&mut self) {
-/// #         println!("Dropping!");
-/// #     }
-/// # }
-/// fn main() {
-///     let _x = HasDrop;
-///
-///     let mut x = _x;
-///     Drop::drop(&mut x);
-/// }
-/// ```
-///
-/// Our custom implementation of `Drop` will be called at the end of `main`.
```

Comment:
Why did you remove these examples?

Reply:
Ah you have more examples below, maybe they are not needed any more.

Reply:
This documentation is already quite long and I think that `This means calling to `Drop::drop` for it, and then recursively for all other values it contains:` + the example with the expected output are hopefully enough